examples/proto_encoding/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/philippgille/gokv/examples/proto_encoding
+
+go 1.18
+
+require (
+	github.com/philippgille/gokv v0.6.0
+	github.com/philippgille/gokv/encoding/proto v0.0.0
+	github.com/philippgille/gokv/gomap v0.6.0
+	google.golang.org/protobuf v1.31.0
+)
+
+replace github.com/philippgille/gokv/encoding/proto => ../../encoding/proto
```

Comment:
:information_source: We can and should remove this once we tag a release and can then import `github.com/philippgille/gokv/encoding/proto` with a proper version.